lib/valkyrie/persistence/postgres/query_service.rb

```diff
@@ -191,7 +191,7 @@ def find_inverse_references_query
     # @return [String]
     def find_references_query
       <<-SQL
```

Comment:
Squiggle heredoc.

Reply:
Why?

Reply:
Just a minor thing to strip the leading whitespace. No significant impact; just tidiness.